teslajsonpy/__init__.py

```diff
@@ -5,61 +5,30 @@
 For more details about this api, please refer to the documentation at
```

Comment:
Please make sure we add the header including the SPDX-License-Identifier to every file.

Reply:
Done.